projects/storefrontlib/shared/components/ng-select-a11y/ng-select-a11y.directive.ts

```diff
@@ -41,6 +43,10 @@ export class NgSelectA11yDirective implements AfterViewInit {
     observer.observe(this.elementRef.nativeElement, { childList: true });
   }
 
+  @Optional() breakpointService = inject(BreakpointService, { optional: true });
+
+  ariaLabelAttribute = 'aria-label';
```

Comment:
I'd be happy with the strings too. However, the Sonarqube is not and gets upset about repeated use :(

Reply:
Ok. Lets move this outside the class then as a constant so it doesn't look like it will change anywhere.

eg.
```
const ARIA_LABEL = 'aria-label';

...

export class ... {
```